src/Redux/Missions/mission.js

```diff
@@ -2,22 +2,59 @@ import { createAsyncThunk } from '@reduxjs/toolkit';
 
 const GET_MISSIONS = 'space-hub/missions/GET_MISSIONS';
 
-const getMissions = createAsyncThunk(GET_MISSIONS, async (args, { dispatch }) => {
-  const response = await fetch('https://api.spacexdata.com/v3/missions');
-  const missions = await response.json();
-  dispatch({
-    type: GET_MISSIONS,
-    payload: missions,
-  });
+const JOINED_MISSION = 'space-hub/missions/JOINED_MISSION';
+const LEFT_MISSION = 'space-hub/missions/LEFT_MISSION';
+
+const getMissions = createAsyncThunk(
+  GET_MISSIONS,
+  async (args, { dispatch }) => {
+    const response = await fetch('https://api.spacexdata.com/v3/missions');
+    const missions = await response.json();
+    dispatch({
+      type: GET_MISSIONS,
+      payload: missions,
+    });
+  },
+);
+
+const joinMission = (mission) => ({
+  type: JOINED_MISSION,
+  payload: mission,
 });
 
-export { getMissions };
+const leaveMission = (mission) => ({
+  type: LEFT_MISSION,
+  payload: mission,
+});
```

Comment:
- [ ]  **Great Job Defining Action Creators! 💯 Please Follow the Duck's Pattern And Put these Actions Creators in the Appropriate ``` profile.js ```**